dbt/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "dbt"
+version = "0.0.1"
+readme = "README.md"
+requires-python = ">=3.11"
+dependencies = [
+  "dbt-athena-community==1.8.2",
+  "dbt-core==1.8.3",
```

Comment:
Note that I basically copied the versions for packages verbatim from the requirements.txt.